### PR TITLE
Bug 1204420 - [Settings] remove I-S panel strings in HTML, r=yzen

### PR DESCRIPTION
**Pull request by gasolin**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=7c8faa042faeeb2ac4fdba4f0dc163fd78e679c3

File: apps/settings/locales/settings.en-US.properties [hunk 1]
```diff
@@ -1452,11 +1452,11 @@ user-guide=User Guide
 default-support=Please contact your local provider for support.
 
 # Device :: SIM toolkit
+simToolkit-header=SIM Toolkit
 operatorServices=Operator Services
 operatorServices-help=Help
-operatorServices-helpmenu=Help menu
+operatorServices-helpmenu=Help Menu
 stkAppsNotAvailable=SIM card applications not available.
-simToolkit=SIM toolkit
```
Comment (gasolin) [inline apps/settings/locales/settings.en-US.properties]: The origin is not used anywhere so I changed it

File: apps/settings/locales/settings.en-US.properties [hunk 2]
```diff
@@ -1452,11 +1452,11 @@ user-guide=User Guide
 default-support=Please contact your local provider for support.
 
 # Device :: SIM toolkit
+simToolkit-header=SIM Toolkit
 operatorServices=Operator Services
 operatorServices-help=Help
-operatorServices-helpmenu=Help menu
```
Comment (gasolin) [inline apps/settings/locales/settings.en-US.properties]: all title item should use Cap case

File: apps/settings/elements/application_storage.html [hunk 3]
```diff
@@ -34,6 +34,7 @@ <h1 data-l10n-id="appStorage-header"></h1>
         </li>
       </ul>
     </div>
+
```
Comment (gasolin) [inline apps/settings/elements/application_storage.html]: to have space between other elements

File: apps/settings/elements/downloads.html [hunk 4]
```diff
@@ -4,10 +4,12 @@
       <h1 data-l10n-id="downloads"></h1>
       <button id="downloads-edit-button" data-icon="edit" data-l10n-id="editButton"></button>
     </gaia-header>
+
     <gaia-header id="downloads-edit-header" action="close" hidden="hidden">
       <h1 id="downloads-title-edit" data-l10n-id="downloads-edit"></h1>
       <button disabled id="downloads-delete-button" data-l10n-id="downloads-delete"></button>
     </gaia-header>
+
```
Comment (gasolin) [inline apps/settings/elements/downloads.html]: to have space between other elements

**Pull request merged**